docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,240 @@
+
+#### a) Steps to create Android Virtual Device:
+1. Install Android Studio from [official link](https://developer.android.com/studio). 
+    > Note: At the time of writing the documentation, the latest version available was Android Studio Giraffe, 2022.3.1 Patch 2 for Window.
+    1. Set ANDROID_HOME environment variable to point to the installation path of Android SDK. i.e. ` C:Users\<user-name>\AppData\Local\Android\Sdk`
+1. Install Java Development Kit (JDK) from [official link](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). For the most recent devices JDK 9 is required, otherwise JDK 8 is required. Make sure you get the JDK and not the JRE.
+    1. Set JAVA_HOME environment variable to the installation path, i.e. ` C:\Program Files\Java\jdk-11`
+1. Create an AVD (Android Virtual Device):
+    1. Open Android Studio. From the Android Studio welcome screen, select **More Action -> Virtual Device Manager**, as instructed [here](https://developer.android.com/studio/run/managing-avds#:~:text=1%20Open%20the%20AVD%20Manager%20by%20clicking%20Tools,%20settings%2C%20such%20as%20the%20skin.%20See%20More.)
+    1. Click **Create Device**.
+    1. Choose a device definition with **Play Store** enabled. This is important, otherwise Teams cannot be installed on the device.
+    1. Choose a System image from the **Recommended** tab which includes access to Google Play services. You may need to install it before selecting it.
+    1. Start the emulator by clicking on the Run button from the Device Manage screen.
+    1. Manually install Microsoft Teams from Google Playstore on the device. 
+
+####  b) Steps to set up Appium
+- Install `appium`:
+    1. Download NodeJs, if it is not already installed on your machine: [Download | Node.js (nodejs.org)](https://nodejs.org/en/download)
+    1. Install Appium globally: [Install Appium - Appium Documentation](https://appium.io/docs/en/2.0/quickstart/install/)
+    1. Install the UiAutomator2 driver: [Install the UiAutomator2 Driver - Appium Documentation](https://appium.io/docs/en/2.0/quickstart/uiauto2-driver/). Go through the `Set up Android automation requirements` in the documentation, to make sure you have set up everything correctly.
+
+        > [Here](https://appium.io/docs/en/2.0/intro/drivers/) you can read more about Appium Drivers.
+    1. Start appium server by running `appium` command in a command prompt.
+####  Useful commands:
+- To list the emulators that you have previously created, without opening Android Studio:
+    ```cli
+    emulator -list-avds
+    ```
+- To start an emulator, without opening Android Studio:
+    ```cli
+    emulator -avd {name-of-emulator}
+    ```
+- To get the name of your AVD, assuming your emulator is running, you can run:
+    
+    ```cli
+    adb devices
+    ```
+    You should see a response with *List of devices attached*, where your running device will be listed.
```

Comment:
```suggestion
####  Useful commands:

List emulators that you have previously created, without opening Android Studio:

```cli
emulator -list-avds
```

Start an emulator, without opening Android Studio:
    
```cli
emulator -avd {name-of-emulator}
```

Get the name of your AVD, assuming your emulator is running, you can run:
    
```cli
adb devices
```

You should see the response *List of devices attached*, where your running device will be listed.
```